README.md

```diff
@@ -138,11 +138,11 @@ Starting up Redis will depend on your operating system, and may in fact already
 
 ## Rails
 
-Hyrax requires Rails 5. We recommend the latest Rails 5.1 release.
+Hyrax 2.x supports Rails 5.1 and 5.2. We recommend the latest Rails 5.2 release.
 
 ```
 # If you don't already have Rails at your disposal...
-gem install rails -v 5.1.6
+gem install rails -v 5.2.4
```

Comment:
How about bumping this to 5.2.4.3 since there have been security patches?

Reply:
ah. it looks like we can do `gem install rails -v '~> 5.2.4'` (or `~>5.2`?) which seems better than both?